small bugfixes

felliot040 opened the pull request:
did some small bug fixes. No changes to gameplay

felliot040 committed:
initial commit take 2

felliot040 committed:
Merge branch 'master' of https://github.com/box-shaped/third-person-controller into Elliot

box-shaped commented:
what stuff specifically did you change
mainly are there any major changes in the player script

box-shaped committed:
Merge branch 'master' into Elliot

felliot040 commented:
line 114 in map, added an underscore infront of TileLibrary, same in line
127 of map.
line 23 in HUD, added an underscore infront of delta
line 14 in monsterSpawner, added an underscore infront of delta and removed
the mapRadius variable from the setup (lines 1-7) because the debug was
complaining that it was "overshadowed" by local variables with the same
name further down the document
Aside from that I don't think I changed anything. All of the above changes
were suggested by warnings in the log.

On Thu, 17 Oct 2024 at 21:03, box-shaped ***@***.***> wrote:

> what stuff specifically did you change
> mainly are there any major changes in the player script
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/box-shaped/third-person-controller/pull/1#issuecomment-2420425979>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLYFCAVIA4KKCNOL7V6743DZ4AJZTAVCNFSM6AAAAABQEMDIROVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMRQGQZDKOJXHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
